extensions/vscode-api-tests/src/singlefolder-tests/notebook.test.ts

```diff
@@ -53,6 +53,10 @@ async function withEvent<T>(event: vscode.Event<T>, callback: (e: Promise<T>) =>
 	await callback(e);
 }
 
+async function applyEditWithRetry(task: vscode.NotebookCellExecutionTask, edit: vscode.NotebookCellOutputEdit) {
+	while (!await task.applyOutputEdit(edit)) { }
```

Comment:
👀 we should really know what's happening before adding this

Reply:
I might not need this anymore...